client/components/loading-spinner.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+class LoadingSpinner extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      loadSuccess: true
+    };
+  }
```

Comment:
I don't believe this constructor is necessary

Reply:
Constructor removed